customer-churn/README.md

```diff
@@ -0,0 +1,247 @@
+
+To configure ECR registry access locally, run, e.g.:
+
+```bash
+aws ecr get-login-password --region us-east-1 | docker login --username AWS \
+  --password-stdin 715803424590.dkr.ecr.us-east-1.amazonaws.com
+```
+
+Extract the URL where the Seldon Core model server exposes its prediction API, e.g.:
+
+```bash
+export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway \
+  -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+```
+
+Configuring the stack can be done like this:
+
+```shell
+zenml integration install s3 aws kubeflow Seldon
+
+zenml artifact-store register aws --type=s3 --path=s3://mybucket
+zenml model-deployer register seldon_aws --type=seldon \
+  --kubernetes_context=zenml-eks --kubernetes_namespace=kubeflow \
+  --base_url=http://$INGRESS_HOST \
+  --secret=s3-store
+zenml container-registry register aws --type=default --uri=715803424590.dkr.ecr.us-east-1.amazonaws.com
+zenml metadata-store register aws --type=kubeflow
+zenml orchestrator register aws --type=kubeflow --kubernetes_context=zenml-eks --synchronous=True
+zenml secrets-manager register aws -t aws
+zenml stack register aws -m aws -a aws -o aws -c aws -d seldon_aws -x aws
+zenml stack set aws
+```
+
+4. Do a pipeline run
+
+```shell
+python run_seldon_deployment_pipeline.py --deploy
+```
+
+You can control which pipeline to run by passing the --deploy and the --predict flag to the `run_seldon_deployment_pipeline.py` launcher. If you run the pipeline with the --deploy flag, the pipeline will train the model and deploy if the model meets the evaluation criteria and then Seldon Core will serve the model for inference. If you run the pipeline with the --predict flag, this tells the pipeline only to run the inference pipeline and not the training pipeline.
+
+You can also set the --min-accuracy to control the evaluation criteria.
+
+5. Configure Port Forwarding and Check the Kubeflow UI to see if the model is deployed and running! 🚀
+
+```bash
+kubectl --namespace kubeflow port-forward svc/ml-pipeline-ui 8080:80
+```
+
+Now, you can go to the localhost:8080 to see the UI.
+
+## 🕹 Demo App [WIP (To be updated)]
+
+**Ignore this section as of now**
+We also made a live demo of this project using [Streamlit](https://streamlit.io/), which you can find [here](https://share.streamlit.io/ayush714/customer-satisfaction/main). Using our trained models takes some input features for the product and predicts the customer satisfaction rate. If you want to run this streamlit app in your local system, you can run the following command:-
+
+```bash
+streamlit run streamlit_app.py
+```
+
+In the inference part of this application, we are fetching our continuous deployment pipeline from the initialized repository, and the pipeline trains the model and (re) deploys the model, and this application makes use of the latest model from the pipeline to predict the customer satisfaction score for the next order or purchase.
+
+## :question: FAQ [WIP]
+
+To be Updated
```

Comment:
Note to complete this...

Reply:
Yeah, I was thinking about your suggestion on making the blog post/guide on these types of question.